src/integration/kotlin/Kafka2Hbase.kt

```diff
@@ -2,16 +2,16 @@ import io.kotlintest.shouldBe
 import io.kotlintest.specs.StringSpec
 import lib.*
 import org.apache.kafka.clients.producer.KafkaProducer
+import java.time.Instant
 
 class Kafka2Hbase : StringSpec({
     configureLogging()
-    
-    val topic = "test-topic".toByteArray()
 
     val producer = KafkaProducer<ByteArray, ByteArray>(Config.Kafka.props)
     val hbase = HbaseClient.connect()
 
     "messages with new identifiers are written to hbase" {
+        val topic = "test-topic-%s".format(Instant.now().toEpochMilli()).toByteArray()
```

Comment:
maybe put this in a function alongside `uniqueBytes` and call it `uniqueTopic`

Reply:
Done